src/main/resources/application.yml

```diff
@@ -1,10 +1,6 @@
 server:
   port: 8081
 
-springfox:
-  documentation:
-    auto-startup: false
-
```

Comment:
Related to the pom file changes above - build issues with updating `private-api-sdk` to a version > 2.0.109.